src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +14,13 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] split = text.replaceAll("[^0-9 -]", " ").split(" ");
```

Comment:
```suggestion
        String[] split = text.replaceAll("[^0-9?-]+", " ").trim().split(" ");
```
this way you won't need additional check inside of for loop

Reply:
got it